core/src/main/scala/not/ogame/bots/package.scala

```diff
@@ -18,4 +18,7 @@ package object bots {
   object FleetId {
     def apply(id: String): FleetId = id.taggedWith[FleetIdTag]
   }
+
+  case class AvailableDeuterExceeded(requiredAmount: BigInt)
```

Comment:
BigInt but why?

Reply:
I though that maximum integer is smaller. I will change it to `Int` then.